DotaStats/Services/NetworkService.swift

```diff
@@ -0,0 +1,38 @@
+import Foundation
+
+protocol NetworkSeviceProtocol: AnyObject {
```

Comment:
Возможно, стоит использовать имя-назначение
```suggestion
protocol NetworkSevice: AnyObject {
```